src/Illuminate/Notifications/Events/BroadcastNotificationCreated.php

```diff
@@ -44,6 +51,10 @@ public function __construct($notifiable, $notification, $data)
         $this->data = $data;
         $this->notifiable = $notifiable;
         $this->notification = $notification;
+
+        if (method_exists($this->notification, 'broadcastNow')) {
+            $this->connection = $this->notification->broadcastNow() ? 'sync' : null;
```

Comment:
Maybe we need to re-think the queue config in 5.4, and have a default "queue queue", and a default "sync queue"?